<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>641</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QFrame" name="frameLeft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>图像属性</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>图像格式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editImg_Format"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>图像宽度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editImg_Width"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>图像高度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="editImg_Height"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>图像深度</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="editImg_Depth"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>位平面数</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="editImg_DPM"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>图像大小</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="editImg_SizeByte"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>DPT分辨率</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="editImg_BitPlane"/>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="chkBox_GrayScale">
           <property name="text">
            <string>灰度图</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="chkBox_Alpha">
           <property name="text">
            <string>具有Alpha通道</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>图像格式转换</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="comboFormat">
           <item>
            <property name="text">
             <string>16-bit RGB565 format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24-bit RGB888 format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32-bit RGB format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8-bit grayscale format</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16-bit grayscale fomat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8-bit indexes into a colomap</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>图像格式转换</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/824.bmp</normaloff>:/images/images/824.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>RGB565数据</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioLSB">
             <property name="text">
              <string>低字节在前</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>高字节在前</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnGetRGB565">
           <property name="text">
            <string>生成RGB565数据</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/116.bmp</normaloff>:/images/images/116.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveDataFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>保存数据文件</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/save.bmp</normaloff>:/images/images/save.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/824.bmp</normaloff>:/images/images/824.bmp</iconset>
      </attribute>
      <attribute name="title">
       <string>图像</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>471</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="labPic">
             <property name="text">
              <string>图片</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/116.bmp</normaloff>:/images/images/116.bmp</iconset>
      </attribute>
      <attribute name="title">
       <string>RGB565</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_Open"/>
   <addaction name="actFile_Reload"/>
   <addaction name="actFile_Save"/>
   <addaction name="actFile_SaveAs"/>
   <addaction name="actFile_Preview"/>
   <addaction name="actFile_Print"/>
   <addaction name="separator"/>
   <addaction name="actImage_ZoomIn"/>
   <addaction name="actImage_ZoomOut"/>
   <addaction name="actImage_RotateLeft"/>
   <addaction name="actImage_RotateRight"/>
   <addaction name="actImage_FilpUD"/>
   <addaction name="actImage_FilpLR"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actFile_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/001.GIF</normaloff>:/images/images/001.GIF</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actFile_Reload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/126.bmp</normaloff>:/images/images/126.bmp</iconset>
   </property>
   <property name="text">
    <string>重新载入</string>
   </property>
   <property name="toolTip">
    <string>重新载入图片</string>
   </property>
  </action>
  <action name="actFile_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/save.bmp</normaloff>:/images/images/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存到当前文件</string>
   </property>
  </action>
  <action name="actFile_SaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/Save-as.ico</normaloff>:/images/images/Save-as.ico</iconset>
   </property>
   <property name="text">
    <string>另存</string>
   </property>
   <property name="toolTip">
    <string>另存图片文件</string>
   </property>
  </action>
  <action name="actFile_Preview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/preview1.bmp</normaloff>:/images/images/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
   <property name="toolTip">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actFile_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/print1.bmp</normaloff>:/images/images/print1.bmp</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="toolTip">
    <string>打印图片</string>
   </property>
  </action>
  <action name="actImage_ZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zoomin.png</normaloff>:/images/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actImage_ZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/zoomout.png</normaloff>:/images/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actImage_RotateLeft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/rotateleft.png</normaloff>:/images/images/rotateleft.png</iconset>
   </property>
   <property name="text">
    <string>左旋</string>
   </property>
   <property name="toolTip">
    <string>左旋</string>
   </property>
  </action>
  <action name="actImage_RotateRight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/rotateright.png</normaloff>:/images/images/rotateright.png</iconset>
   </property>
   <property name="text">
    <string>右旋</string>
   </property>
   <property name="toolTip">
    <string>右旋</string>
   </property>
  </action>
  <action name="actImage_FilpUD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/flip-vertical.ico</normaloff>:/images/images/flip-vertical.ico</iconset>
   </property>
   <property name="text">
    <string>上下翻转</string>
   </property>
   <property name="toolTip">
    <string>上下翻转</string>
   </property>
  </action>
  <action name="actImage_FilpLR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/flip-horizontal.ico</normaloff>:/images/images/flip-horizontal.ico</iconset>
   </property>
   <property name="text">
    <string>左右翻转</string>
   </property>
   <property name="toolTip">
    <string>左右翻转</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
